src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -2696,10 +2696,16 @@ static void _inheritGradient(SvgLoaderData* loader, SvgStyleGradient* to, SvgSty
             //GradUnits not set directly, coord set
             if (!gradUnitSet && coordSet) {
                 radialTags[i].tagRecalc(loader, to->radial, to->userSpace);
+                //If fx and fy are not set, set cx and cy.
+                if (!strcmp(radialTags[i].tag, "cx") && (to->flags & SvgGradientFlags::Cx) && !(to->flags & SvgGradientFlags::Fx)) to->radial->fx = to->radial->cx;
```

Comment:
the center position is mandatory in radial gradient. In the exceptional case, it would overwrite with the default value anyway. Thus this condition looks unnecessary. 
``` !strcmp(radialTags[i].tag, "cx") && (to->flags & SvgGradientFlags::Cx) ```

Reply:
um.. depending on the inheritance status, there may be a case where cx or cy is missing, but it will probably be overwritten with 0.
I removed `(to->flags & SvgGradientFlags::Cx)` condition.
`strcmp()` should be necessary. If not, it is overwritten at every tag, so recalculated fx or ,fy may be overwritten in `fr`.